portal-web/test/functional/com/liferay/portalweb/paths/portlet/questions/Questions.path

```diff
@@ -53,6 +53,11 @@
 	<td>//select[@id='_com_liferay_message_boards_web_portlet_MBAdminPortlet_displayStyle']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>DELETE_QUESTION</td>
```

Comment:
@liuxingye can we reuse a general locator in Button.path here?

Reply:
Use Icon#DELETE instead of it.